src/client/eppo-client.ts

```diff
@@ -984,7 +983,15 @@ export default class EppoClient {
   }
 
   public useLRUInMemoryBanditAssignmentCache(maxSize: number) {
-    this.banditAssignmentCache = new LRUInMemoryAssignmentCache(maxSize);
+    this.banditAssignmentCache = new TLRUInMemoryAssignmentCache(maxSize);
+  }
+
+  /**
+   * @param {number} maxSize - Maximum cache size
+   * @param {number} timeout - TTL of cache entries
+   */
+  public useTLRUInMemoryAssignmentCache(maxSize: number, timeout?: number) {
```

Comment:
This should be axed and timeout added to the previous method. We only want the expiring option for bandits